xml/System.Data.SqlClient/SqlException.xml

```diff
@@ -49,7 +49,7 @@
  So, in general, write an exception handler that catches any provider specific exceptions as well as exceptions from the common language runtime.  These can be layered as follows:  
   
 ```  
```

Comment:
would you mind also adding the code block identifier (csharp) here and on the next code block so the code is also properly colorized?

Reply:
Certainly. I’ll get to that shortly.

Reply:
@mairaw done.